### PR TITLE
Add `Number.isInteger` polyfill

### PR DESCRIPTION
**Pull request by avocadowastaken**

### CODE REVIEW

File: polyfills/Number/isInteger/tests.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  proclaim.equal(Number.isInteger.length, 1);
+});
+
+it('returns true with integer values', function () {
+  proclaim.equal(Number.isInteger(0), true);
+  proclaim.equal(Number.isInteger(1), true);
+  proclaim.equal(Number.isInteger(-100000), true);
+});
+
+it('retuns false for non integer values values', function () {
```
Comment (rowanmanning) [inline polyfills/Number/isInteger/tests.js]: typo: "values values"

**Review by rowanmanning — CHANGES_REQUESTED** [review]
Looks great, thanks @umidbekkarimov. One small typo but otherwise happy to merge

**Comment by avocadowastaken** [comment]
@rowanmanning oops, was in little rush this morning, fixed 👍

**Review by rowanmanning — APPROVED** [review]

File: polyfills/Number/isInteger/config.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+	"aliases": [
+		"es6"
+	],
+	"browsers": {
+		"ie": "*",
```
Comment (rowanmanning) [inline polyfills/Number/isInteger/config.json]: The browsers list isn't quite complete, [we maintain a list of supported ones here](https://polyfill.io/v2/docs/contributing/authoring-polyfills#browsers). The [Kangax compatibility tables](kangax.github.io/compat-table/es6) are a good place to get this information.

**Review by rowanmanning — COMMENTED** [review]
You'll also need to [accept our contribution terms](https://polyfill.io/v2/docs/contributing#contribution-terms). Written agreement here in the PR would be fine

**Comment by avocadowastaken** [comment]
Any contribution I make to the polyfill service by way of Pull Request contains only data that is:
- wholly my own work, upon which I am legally permitted to grant a licence; or
- wholly the work of individuals within the same organisation to which the copyright in all work created by employees is assigned, of which I am a current employee and authorised representive for the purposes of this contribution, and which is legally permitted to grant a licence; or
- licensed by the original author under the MIT licence or equivalent terms

In respect of any part of my contribution that is not already available under an MIT or compatible licence, I hereby grant to the Financial Times Limited (FT) and to recipients of software distributed by the FT a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable license to make, have made, use, offer to sell, sell, import, and otherwise transfer the data.

**Comment by avocadowastaken** [comment]
@rowanmanning done and done

**Comment by JakeChampion** [comment]
Great work @umidbekkarimov! 

Did you know we offer one month's free premium access to all FT content (normally $50) to anyone who has a pull request merged in this repository? If you want to claim it, there's [one Google form](https://docs.google.com/a/ft.com/forms/d/1aykPiYBwqKpKY6nvGVi43wOU_5epr3XLgE00Vi52SMQ/viewform) to fill out :-)

**Pull request merged**